package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "spline",
-  "version": "1.0.1",
-  "description": "A set of Sass utility mixins for Mobify's client services team",
-  "main": "dist/_spline.scss",
+  "name": "spline-scss",
+  "version": "2.0.0",
+  "description": "A set of Sass utility mixins and functions for Mobify's client services team",
+  "main": "dist/spline.scss",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "sass-lint -c node_modules/mobify-code-style/css/sass-lint.yml **/*.scss -v"
```

Comment:
A couple recommendations for this linting command:
- The local version of the sass-lint bin should be invoked instead, since some users may have a different version of `sass-lint` installed on their machine
- The `**/*.scss` glob should be in quotes, so that all errors from all matched files appear together

My recommended linting command:
`./node_modules/.bin/sass-lint -c node_modules/mobify-code-style/css/.sass-lint.yml '**/*.scss' -v`